BuildAndTest.cmd

```diff
@@ -5,52 +5,82 @@ SETLOCAL
+echo {                                                                               >> %VERSION_CONSTANTS%
+echo     public static class VersionConstants                                        >> %VERSION_CONSTANTS%
+echo     {                                                                           >> %VERSION_CONSTANTS%
+echo         public const string Prerelease = "%PRERELEASE%";                        >> %VERSION_CONSTANTS%
 echo         public const string AssemblyVersion = "%MAJOR%.%MINOR%.%PATCH%" + ".0"; >> %VERSION_CONSTANTS%
-echo         public const string FileVersion = "%MAJOR%.%MINOR%.%PATCH%" + ".0";    >> %VERSION_CONSTANTS%
-echo         public const string Version = AssemblyVersion + Prerelease;            >> %VERSION_CONSTANTS%
-echo     }                                                                          >> %VERSION_CONSTANTS%
-echo  }                                                                            >> %VERSION_CONSTANTS%
+echo         public const string FileVersion = "%MAJOR%.%MINOR%.%PATCH%" + ".0";     >> %VERSION_CONSTANTS%
+echo         public const string Version = AssemblyVersion + Prerelease;             >> %VERSION_CONSTANTS%
+echo     }                                                                           >> %VERSION_CONSTANTS%
+echo  }                                                                              >> %VERSION_CONSTANTS%
+
+%~dp0.nuget\NuGet.exe restore src\BinSkim.sln -ConfigFile "%NuGetConfigFile%" -OutputDirectory "%NuGetPackageDir%"
 
-%~dp0.nuget\NuGet.exe restore src\BinSkim.sln 
-msbuild /verbosity:minimal /target:rebuild src\BinSkim.sln /p:Configuration=Release /p:"Platform=x64" /filelogger /fileloggerparameters:Verbosity=detailed || goto :ExitFailed
-msbuild /verbosity:minimal /target:rebuild src\BinSkim.sln /p:Configuration=Release /p:"Platform=x86" /filelogger /fileloggerparameters:Verbosity=detailed || goto :ExitFailed
+:: Build the solution 
+dotnet clean /verbosity:minimal src\BinSkim.sln /p:Configuration=%Configuration% || goto :ExitFailed
+dotnet build /verbosity:minimal src\BinSkim.sln /p:Configuration=%Configuration% /filelogger /fileloggerparameters:Verbosity=detailed || goto :ExitFailed
 
-md bld\bin\nuget
-md bld\bin\LayoutForSigning
-md bld\bin\LayoutForSigning\x86
-md bld\bin\LayoutForSigning\x64
+::Run unit tests
+echo Run all multitargeting xunit tests
+call :RunMultitargetingTests Driver Functional || goto :ExitFailed
+call :RunMultitargetingTests Rules Functional  || goto :ExitFailed
 
-xcopy /Y bld\bin\x86_Release\BinSkim.exe       bld\bin\LayoutForSigning\x86
-xcopy /Y bld\bin\x86_Release\BinaryParsers.dll bld\bin\LayoutForSigning\x86
-xcopy /Y bld\bin\x86_Release\BinSkim.Rules.dll bld\bin\LayoutForSigning\x86
-xcopy /Y bld\bin\x86_Release\BinSkim.Sdk.dll   bld\bin\LayoutForSigning\x86
+::Create the BinSkim platform specific publish packages
+echo Creating Platform Specific BinSkim 'Publish' Packages

```

Comment:
All I see in my bin folder are build output for each framework. No platform-specific folders.